README.md

```diff
@@ -34,10 +34,6 @@ rules_sass_dependencies()
 # Setup repositories which are needed for the Sass rules.
 load("@io_bazel_rules_sass//:defs.bzl", "sass_repositories")
 sass_repositories()
-
-# Setup the NodeJS toolchain
-load("@build_bazel_rules_nodejs//:defs.bzl", "node_repositories")
```

Comment:
`node_repositories()` is no longer explicitly since `sass_repositories()` calls `yarn_install` which will call `node_repositories()` if it has not already been called.